source/news/2019-10-03.html.markdown

```diff
@@ -112,7 +112,7 @@ class CreateUser
 end
 ```
 
-Here `CreateUser` is not linked to dependency implementation in any way. To provide the dependency add a handler:
+Here `CreateUser` is not linked to a dependency resolution implementation in any way. To provide the dependency, add a handler:
```

Comment:
I _think_ this might have clarified it a little? But I could've been wrong about your technical meaning here, so again, happy for you to adjust :)